changelog.md

```diff
@@ -1,5 +1,151 @@
+- Add: [NewGRF] Provide random bits in var10 of house callback 1C 'construction stage changed'. (#14095)
+- Add: [NewGRF] Station/roadstop animation-triggers 'tile loop' (bit 7) and 'path reservation' (bit 8). (#14080)
+- Change: Improve sprite ellipsis appearance. (#14404)
+- Change: New company face definition system and UI. (#14319)
+- Change: Simplify sprite cache memory management
+- Change: Tweak layout of music player widgets.
+- Change: Consistent toolbar sizes with dedicated NWidgetParts. (#14344)
+- Change: Reflow Textfile window content incrementally.
+- Change: Don't measure group name widths in company livery window. (#14291)
+- Change: [Script] Don't allow scripts to instantiate ScriptEvent (#14276)
+- Change: [Script] ScriptVehicleList_Station accepts an optional VehicleType parameter (#14260)
+- Change: Remove Apply button from NewGRF config window when unneeded. (#14254)
+- Change: Remove extra close buttons from some windows. (#14124)
+- Change: Move OpenTTD version to intro viewport. (#14233)
+- Change: [NewGRF] Increase the textstack for all callbacks to 16 registers.
+- Change: Rearrange buttons in the intro screen, remove unnecessary ones.
+- Change: Remove the limit of 20 parameters to a Script Text string. (#14193)
+- Change: Use same padding as NewGRF parameters for AI/GS parameters lists. (#14118)
+- Change: ParseInteger now checks for trailing junk chars.
+- Change: [NewGRF] If Action123 does not resolve in a valid SpriteSet, prefer drawing the default sprite instead of an invalid sprite.
+- Change: Draw info headers as shade of window colour instead of pixel colour. (#14123)
+- Change: [NewGRF] Animation-trigger 'construction stage changed' of houses and industries now also triggers at construction start. (#14089)
+- Change: [Script] Reject scripts using negative version (#14096)
+- Change: [NewGRF] Provide shared random bits in multi-tile animation-triggers of airport tiles and objects, just like for other features. (#14090)
+- Change: Move Game Options toggles to before label, add Off/On text. (#14097)
+- Change: Remove minimum width from advanced settings panel of Game Options window. (#14088)
+- Change: Use standard mauve/yellow scheme for game options window.
+- Change: Adjust section names in Graphics options tab.
+- Change: Merge Game Options and Game Settings together.
+- Change: Enhance visibility of disabled bool button. (#14071)
+- Change: Use scaled instead of fixed dimensions for framerate graphs. (#14039)
+- Change: Include dragged train in depot tile length display. (#14060)
+- Change: Use standard boolean toggle buttons for game options toggles.
+- Change: Draw boolean toggle as a slider widget.
+- Change: "setting" console command now shows default value (#14061)
+- Change: Allow Double-Ctrl+Click on default size box to clear saved size. (#14055)
+- Change: Don't select content when toggling its download status. (#14059)
+- Change: Add scrollbar to infrastructure window. (#14056)
```

Comment:
Players have been asking for this for a while, let's make it Add (or maybe even Feature 😛 ) for visibility.

```suggestion
- Add: Add scrollbar to infrastructure window. (#14056)
```